FizzBuzz/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*FizzBuzz
+ * A program that writes numbers from 1 to 100. Any number that is a multiple of three is replaced
+ * by the word 'Fizz', any number that is a multiple of five is replaced by the word 'Buzz', and any
+ * number that is a multiple of both three and five is replaced by the word 'FizzBuzz'.
+ */
+
+using System;
+
+namespace FizzBuzz
+{
+    class Program
+    {
+            static void Main(string[] args)
+            {
+                double ii;
+
+
+                for (ii = 1; ii <= 100; ii += 1)
+                {
+                    double div1 = (ii / 3);
```

Comment:
I like the idea of storing the values here, but it comes at the cost or reducing readability - I have to remember that you are dividing the number here and figure out the value you are using further below when you use the modulo. Can we improve readability overall by create a function that take the current value ii, the number we are checking against (3 or 5), and returns whether the value ii is a multiple